src/ws/protocol.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * TODO: use types from graasp-websockets
+ */
+import {
+  CLIENT_ACTION_DISCONNECT,
+  CLIENT_ACTION_SUBSCRIBE,
+  CLIENT_ACTION_SUBSCRIBE_ONLY,
```

Comment:
I guess it's copy-pasted from query-client. Should we put that in sdk? Or in graasp-websockets?

Reply:
See https://github.com/graasp/graasp-sdk/tree/main/src/services/websockets/api